WalletWasabi.Fluent/Controls/ContentArea.axaml

```diff
@@ -62,7 +62,7 @@
 
                         <Panel DockPanel.Dock="Top">
                             <Panel IsHitTestVisible="False" Background="{TemplateBinding HeaderBackground}" />
-                            <StackPanel Name="PART_Header" Spacing="10">
+                            <StackPanel Name="PART_Header" Height="32" Spacing="10">
```

Comment:
This broke the layout, almost everywhere. Do not modify the ContentArea, it is very risky.
Rather just add the modification as a style in the welcomepage view.

Reply:
Done!